backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5726940_sys_gh18285_Use_Lookup_Instead_Listview.sql

```diff
@@ -0,0 +1,23 @@
+select backup_table('ad_column'); -- backup.ad_column_bkp_20240620_114638_818
+select backup_table('ad_field'); -- backup.ad_field_bkp_20240620_114658_431
+
+UPDATE AD_Column
+SET AD_Reference_ID = 30
+WHERE AD_Reference_ID IN (18, 19)
+;
+
+UPDATE AD_Field
+SET AD_Reference_ID = 30
+WHERE AD_Reference_ID IN (18, 19)
+;
+
+UPDATE AD_UI_Element
+SET UIStyle=''
```

Comment:
why?

Reply:
i found a problem in the Doc Type Target field in salse order window, that have UIStyle='secondary' so even the reference was set to searche the field still displayed as list

Reply:
we would need an impact analysis.... preciselly which are the fields affected

Reply:
also, is that logic coming from backend or frontend?

AIM: we need to understand what's the best, 
1. update all AD_UI_Elements blindly like a tsunami
2. understand what's the logic behind it and maybe change that...

Reply:
ok, skip impact on UI side